onnx/defs/logical/defs.cc

```diff
@@ -219,4 +219,99 @@ ONNX_OPERATOR_SET_SCHEMA(
         }
         )ONNX"));
 
+std::function<void(OpSchema&)> BinaryBitwiseDocGenerator(const char* name) {
+  return [=](OpSchema& schema) {
+    std::string doc;
+    POPULATE_OP_DOC_STR(doc = R"DOC(
+Returns the tensor resulted from performing the `{name}` bitwise operation
```

Comment:
Nit:
(i) change `resulted` to `resulting`. 
(ii) exchange the order of `{name}` and `bitwise`.

Reply:
Good catch! That is definitely better wording

I've made the changes to the PR